docs/interpreter/cassandra.md

```diff
@@ -1,4 +1,4 @@
----
+z---
```

Comment:
why this change?

Reply:
Just typo, didn't notice when committing.